kdestandardize.ado

```diff
@@ -1399,18 +1412,51 @@ prog def kdestandardize
 									
 	} // End of handling of the Equity_Measure  variable
 	
-	// Handles instances of the  variable
-	if `: list posof "" in x' != 0 {
-	
-	} // End of handling of the  variable
-	
-	// Handles instances of the  variable
-	if `: list posof "" in x' != 0 {
-	
-	} // End of handling of the  variable
-	
-	
-	  
+	// Handles instances of the EQ_LABEL variable
+	if `: list posof "EQ_LABEL" in x' != 0 {
+		qui: replace EQ_LABEL=cond(`rx'(EQ_LABEL, "Exemplary/ Accomplished", 1), "1", ///
+							  cond(EQ_LABEL=="High/Expect","2",              ///
+							  cond(EQ_LABEL=="School-Level","3",             ///
+							  cond(EQ_LABEL=="Strongly Agree/Agree","4"      ///
+							  cond(EQ_LABEL==" ","5")))))                    ///
+		qui: rename EQ_LABEL eqlabel
+		qui: destring eqlabel,replace ignore("*,-R %")
+		la val eqlabel eqlabel
+		la var eqlabel Equity Label
+	} // End of handling of the EQ_LABEL variable
+	
+	// Handles instances of the EQ_PCT variable
+	if `: list posof "EQ_PCT" in x' != 0 {
+	qui: rename EQ_PCT eqpct
+		qui: destring eqpct, replace ignore("*,-R %")
+		la var eqpct "Equity Percent"
+	} // End of handling of the EQ_PCT variable
+	
+	// Handles instances of the PROGRAM_TYPE variable
+	if `: list posof "PROGRAM_TYPE" in x' != 0 {
+	qui: replace PROGRAM_TYPE=cond(`rx'(PROGRAM_TYPE, "English Language Learners (ELL)", 1), "1", ///
+							  cond(PROGRAM_TYPE=="Migrant","2",              ///
+							  cond(PROGRAM_TYPE=="Special Education","3",    ///
+							  cond(PROGRAM_TYPE=="Gifted and Talented","4")))) ///
+	qui: rename PROGRAM_TYPE prgmtp
+	qui: desting prgmtp,replace ignore("*,-R %")
+	la val prgmtp prgmtp
+	la var prgmtp Program Type
+	} // End of handling of the PROGRAM_TYPE variable
+	
+	// Handles instances of the TOTAL_CNT variable
```

Comment:
For cases where `total` should be referenced in the variable name use `tot` instead.  In this case using `membership` would probably be better and more consistent with other reporting standards for student counts by demographic groups/special programs.